SU-SWS/linky_clicky
# Add tests for planning.stanford.edu

# READY FOR REVIEW

# Summary
- Tests for verifying authenticated/anonymous user access on planning.stanford.edu

# Needed By (Date)
- Next week maybe?

# Criticality
- Not super critical. Affects only one site.

# Steps to Test

1. Check out this branch
2. Run `sites/lrp/features/content.feature` against planning.stanford.edu

# Affected Projects or Products
- planning.stanford.edu

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/LRP17-53
- Anyone who should be notified? ( @annawatt )
